frontend/src/lib/contexts/ExtraConfigContext.ts

```diff
@@ -0,0 +1,16 @@
+import { createContext } from "react";
+
+export type OidcInfo = {
+  enabled: boolean;
+  loginUrl: string;
+};
+
+export type ExtraConfigContextType = {
```

Comment:
Can we make this specific to just OidcInfo?  I'd like to prevent arbitrary stuff being put into a context provider.